Controller/Api/GroupController.php

```diff
@@ -61,22 +61,35 @@ public function __construct(GroupManagerInterface $groupManager, FormFactoryInte
      *
      * @ApiDoc(
      *  resource=true,
-     *  output={"class"="Sonata\DatagridBundle\Pager\PagerInterface", "groups"="sonata_api_read"}
+     *  output={"class"="Sonata\DatagridBundle\Pager\PagerInterface", "groups"={"sonata_api_read"}}
```

Comment:
nothing with fos-rest, i just wanted to uniform the annotations inside the controllers: some were arrays and other strings. Also, nelmio-apidoc usually expects groups as an array, and they convert it in ApiDocExtractor with this
`// normalize groups`
`if (isset($input['groups']) && is_string($input['groups'])) {`
`   $input['groups'] = array_map('trim', explode(',', $input['groups']));`
`}`

so, given the array need of fos-rest, this also seemed right to do to prevent further work ahead in case they deprecate that one. It does not break BC in any way.